packages/use-wallet/src/index.ts

```diff
@@ -1,7 +1,7 @@
-import { useCallback } from 'react';
-import Web3Modal, { ICoreOptions } from 'web3modal';
-import { Network, Web3Provider } from '@ethersproject/providers';
-import create from 'zustand';
+import { useCallback } from "react";
```

Comment:
Why is `useWallet` using `"` for string?

Reply:
Good catch, added a `.prettierrc` for that package!